src/wp-includes/class-wp-duotone.php

```diff
@@ -948,45 +944,42 @@ public static function register_duotone_support( $block_type ) {
 	 * This handles selectors defined in `color.__experimentalDuotone` support
 	 * if `filter.duotone` support is not defined.
 	 *
-	 * @param string $block_name The block name.
-	 *
 	 * @internal
-	 *
 	 * @since 6.3.0
-	 *
+	 * @param WP_Block_Type $block_type Block type to check for support.
```

Comment:
Whoops now there is no empty line before this one. Apologies if I missed it in my suggestion.

Too bad PHPCS doesn't properly flag this on PRs

Reply:
```suggestion
	 *
	 * @param WP_Block_Type $block_type Block type to check for support.
```